### ISSUE TITLE
Invalid value 'SG_VOEC' for 'deemed_reseller_category'

### ISSUE THREAD

**Issue by quazardous** [jlevers/selling-partner-api#466]
The order is legit but got an error.
```
InvalidArgumentException: Invalid value 'SG_VOEC' for 'deemed_reseller_category', must be one of 'IOSS', 'UOSS', 'GB_VOEC', 'NO_VOEC', 'CA_MPF', 'AU_VOEC', 'NZ_VOEC' in /x/y/z/vendor/jlevers/selling-partner-api/lib/Model/OrdersV0/OrderItem.php:1123
```

**Comment by quazardous** [comment]
Maybe a less restrictive strategy could be implemented for new values like some warning instead of a fatal exception.

**Comment by apolitano1** [comment]
The value: `KZ_VOEC` also throws this error

**Comment by jlevers** [comment]
This should be fixed in `v5.5.3`. Let me know if you continue to run into issues.
